domains/certificates/Query.API/API.IntegrationTests/CertificateIssuingTests.cs

```diff
@@ -63,7 +64,32 @@ public async Task MeasurementsSyncerSendsMeasurementsToStamp_ExpectInWallet()
                     }
                 }
             });
+
         factory.measurementsClient = measurementClientMock;
+        var meteringpointClientMock = Substitute.For<Meteringpoint.V1.Meteringpoint.MeteringpointClient>();
+
+        var mockedMeteringPointsResponse = new Meteringpoint.V1.MeteringPointsResponse
+        {
+            MeteringPoints =
+            {
+                new Meteringpoint.V1.MeteringPoint
+                {
+                    MeteringPointId = gsrn,
+                    MeteringPointAlias = "alias",
+                    ConsumerStartDate = "consumerStartDate",
+                    Capacity = "123",
+                    BuildingNumber = "buildingNumber",
+                    CityName = "cityName",
+                    Postcode = "postcode",
+                    StreetName = "streetName",
+                }
+            }
+        };
```

Comment:
Why does meteringpoints use grpc now instead of rest? What about the frontende?

Reply:
It is a internal call

Reply:
but what about the frontend? doesent that use this?

Reply:
No the frontend wont need this